pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1070,6 +1070,14 @@
             + " and resolving its metadata service location"
     )
     private String bookkeeperMetadataServiceUri;
+    @FieldContext(
+        category = CATEGORY_STORAGE_BK,
+        doc = "With Opportustic Striping the bookeeper client will fallback to use less bookies in case "
```

Comment:
A few typos, fixed:
"With Opportunistic Striping the bookkeeper client will fall back to use fewer bookies in case "

Reply:
thank you @lhotari 
I have fixed this line according to your suggestion